fossology/fossology
# feat(scanners) : ignore files from scanning using mimetype

Signed-off-by: Anupam Ghosh <anupam.ghosh@siemens.com>



## Description

Sometime scanner ( like nomos takes longtime to finish),
Idea is to skip files with particular mimetypes those not require eg : .class , image, doc, binary  

### Changes

New field in Admin-->Customize --> "Skip MimeTypes from scanning"
   -- to add mimetypes of the files need to be included 

User can decide to include or not to include the mimetype/s included in Admin-->customized page 
(using the checkbox "Ignore SCM files (Git, SVN, TFS) and files with particular Mimetype" in upload page) 


## How to test

1. Add mimetypes that need to be excluded by scanner(now only for nomos) 
in Admin-->Customize --> "Skip MimeTypes from scanning
1. check the checkbox on upload page "Ignore SCM files (Git, SVN, TFS) and files with particular Mimetype"
1. include nomos from scanner list
(https://help.github.com/articles/closing-issues-using-keywords/)
